src/components/common/text/FiatText.js

```diff
@@ -1,52 +1,30 @@
 // @flow
-import { div } from 'biggystring'
 
-import { getSymbolFromCurrency } from '../../../constants/WalletAndCurrencyConstants'
-import { convertCurrency } from '../../../selectors/WalletSelectors'
-import { useSelector } from '../../../types/reactRedux'
-import { DECIMAL_PRECISION, formatFiatString } from '../../../util/utils'
+import { useFiatText } from '../../../hooks/useFiatText'
+
+/**
+ * - raw: Only the numeric fiat text itself
+ * - primary: Fiat text with the fiat symbol
+ * - secondary: Fiat text enclosed in parenthesis, used when the fiat text is
+ * combined with a crypto amount
+ **/
+export type FiatTextFormat = 'raw' | 'primary' | 'secondary'
```

Comment:
This came down from @thehobbit85 but I like this better as well

Reply:
@swansontec This isn't going to work. I can't use `FiatText` as a JSX component without adding some ugly `<EdgeText>` around each parenthesis. If I use `FiatText` as a method, everything class-based explodes since I need to add a `useSelector` to get the `account` inside of `FiatText` to satisfy the other comments about handling the `tokenId`.

Reply:
Go to `.eslintrc.json` and edit the rule to say `"react-native/no-raw-text": ["error", { "skip": ["FormattedText", "T", "B", "EdgeText"] }]`. Adding `EdgeText` to the list should make the usage legal.

However, I am OK making a boolean prop called `parenthesisEnclosed`. It's a better solution than the enum.